modules/textures/src/lib/parsers/parse-basis.js

```diff
@@ -29,6 +30,14 @@ const OutputFormat = {
  * @returns compressed texture data
  */
 export default async function parseBasis(data, options) {
+  if (options.basis.decoderFormat === 'auto') {
+    if (isKTX(data)) {
+      const fileConstructors = await loadBasisEncoderModule(options);
+      return parseKTX2File(fileConstructors.KTX2File, data, options);
```

Comment:
Does this handle the case when the file is basis without KTX? Can you add some comments and JSDoc?

Side note: I wonder if we would be able to create a `KTX2Loader`?

Reply:
It does